tests/Backend/Bigquery/TimeTravelTest.php

```diff
@@ -0,0 +1,288 @@
+<?php
+
+namespace Keboola\Test\Backend\Bigquery;
+
+use Keboola\Csv\CsvFile;
+use Keboola\StorageApi\Client;
+use Keboola\StorageApi\ClientException;
+use Keboola\StorageApi\Options\TokenAbstractOptions;
+use Keboola\StorageApi\Options\TokenCreateOptions;
+use Keboola\StorageApi\Options\TokenUpdateOptions;
+use Keboola\StorageApi\TableExporter;
+use Keboola\Test\StorageApiTestCase;
+
+class TimeTravelTest extends StorageApiTestCase
```

Comment:
To je C&p toho SNFLK testu. Nebylo by lepší ho vycuknůt někam společně?

Reply:
@zajca chcel som to prave nejak spolocne ale su tam nejake rozdiely, plus teraz v BQ pribudnu testy s datovymi typami, takze to bude uz trochu rozdielne ma to myslis stale cenu nejak spojovat ?

Reply:
Ale asi hej tak ja to nejak spojim :)